diff --git a/m1.py b/m1.py
index 2a55171..7328a80 100644
--- a/m1.py
+++ b/m1.py
@@ -1 +1,2 @@
 v=1
+extra=1
diff --git a/m10.py b/m10.py
index 0d7be67..661ca52 100644
--- a/m10.py
+++ b/m10.py
@@ -1 +1,2 @@
 v=10
+extra=1
diff --git a/m11.py b/m11.py
index cbc715d..01673d5 100644
--- a/m11.py
+++ b/m11.py
@@ -1 +1,2 @@
 v=11
+extra=1
diff --git a/m12.py b/m12.py
index 7e7a406..10f41dc 100644
--- a/m12.py
+++ b/m12.py
@@ -1 +1,2 @@
 v=12
+extra=1
diff --git a/m13.py b/m13.py
index 972203a..39d64bb 100644
--- a/m13.py
+++ b/m13.py
@@ -1 +1,2 @@
 v=13
+extra=1
diff --git a/m14.py b/m14.py
index c06e7a0..f59cd60 100644
--- a/m14.py
+++ b/m14.py
@@ -1 +1,2 @@
 v=14
+extra=1
diff --git a/m15.py b/m15.py
index 23784fd..682f8ff 100644
--- a/m15.py
+++ b/m15.py
@@ -1 +1,2 @@
 v=15
+extra=1
diff --git a/m16.py b/m16.py
index 62f2ffe..61a7798 100644
--- a/m16.py
+++ b/m16.py
@@ -1 +1,2 @@
 v=16
+extra=1
diff --git a/m17.py b/m17.py
index cb69cb7..78620b5 100644
--- a/m17.py
+++ b/m17.py
@@ -1 +1,2 @@
 v=17
+extra=1
diff --git a/m18.py b/m18.py
index 8387f5d..076985d 100644
--- a/m18.py
+++ b/m18.py
@@ -1 +1,2 @@
 v=18
+extra=1
diff --git a/m19.py b/m19.py
index ab36294..880159c 100644
--- a/m19.py
+++ b/m19.py
@@ -1 +1,2 @@
 v=19
+extra=1
diff --git a/m2.py b/m2.py
index 8c6126c..9933ceb 100644
--- a/m2.py
+++ b/m2.py
@@ -1 +1,2 @@
 v=2
+extra=1
diff --git a/m20.py b/m20.py
index ffdb439..566265e 100644
--- a/m20.py
+++ b/m20.py
@@ -1 +1,2 @@
 v=20
+extra=1
diff --git a/m21.py b/m21.py
index ef8ff9a..2b04637 100644
--- a/m21.py
+++ b/m21.py
@@ -1 +1,2 @@
 v=21
+extra=1
diff --git a/m22.py b/m22.py
index 8479c48..74317b5 100644
--- a/m22.py
+++ b/m22.py
@@ -1 +1,2 @@
 v=22
+extra=1
diff --git a/m23.py b/m23.py
index b7cb716..8c8f8bc 100644
--- a/m23.py
+++ b/m23.py
@@ -1 +1,2 @@
 v=23
+extra=1
diff --git a/m24.py b/m24.py
index 8107d44..ff1d887 100644
--- a/m24.py
+++ b/m24.py
@@ -1 +1,2 @@
 v=24
+extra=1
diff --git a/m25.py b/m25.py
index 32f4bc8..98e7810 100644
--- a/m25.py
+++ b/m25.py
@@ -1 +1,2 @@
 v=25
+extra=1
diff --git a/m26.py b/m26.py
index 1ed983c..b08ce56 100644
--- a/m26.py
+++ b/m26.py
@@ -1 +1,2 @@
 v=26
+extra=1
diff --git a/m27.py b/m27.py
index 8929fc0..b641fea 100644
--- a/m27.py
+++ b/m27.py
@@ -1 +1,2 @@
 v=27
+extra=1
diff --git a/m28.py b/m28.py
index 5963c0c..8e55643 100644
--- a/m28.py
+++ b/m28.py
@@ -1 +1,2 @@
 v=28
+extra=1
diff --git a/m29.py b/m29.py
index 90df66a..f6d41fd 100644
--- a/m29.py
+++ b/m29.py
@@ -1 +1,2 @@
 v=29
+extra=1
diff --git a/m3.py b/m3.py
index fd509a5..5121671 100644
--- a/m3.py
+++ b/m3.py
@@ -1 +1,2 @@
 v=3
+extra=1
diff --git a/m30.py b/m30.py
index 33a4993..1687ac7 100644
--- a/m30.py
+++ b/m30.py
@@ -1 +1,2 @@
 v=30
+extra=1
diff --git a/m31.py b/m31.py
index dcc3403..0703363 100644
--- a/m31.py
+++ b/m31.py
@@ -1 +1,2 @@
 v=31
+extra=1
diff --git a/m32.py b/m32.py
index a1aeaf9..4b40117 100644
--- a/m32.py
+++ b/m32.py
@@ -1 +1,2 @@
 v=32
+extra=1
diff --git a/m33.py b/m33.py
index e3079f6..dae82fd 100644
--- a/m33.py
+++ b/m33.py
@@ -1 +1,2 @@
 v=33
+extra=1
diff --git a/m34.py b/m34.py
index c80be07..1d06782 100644
--- a/m34.py
+++ b/m34.py
@@ -1 +1,2 @@
 v=34
+extra=1
diff --git a/m35.py b/m35.py
index f275ece..751f6f2 100644
--- a/m35.py
+++ b/m35.py
@@ -1 +1,2 @@
 v=35
+extra=1
diff --git a/m36.py b/m36.py
index b9fc018..dc47bc2 100644
--- a/m36.py
+++ b/m36.py
@@ -1 +1,2 @@
 v=36
+extra=1
diff --git a/m37.py b/m37.py
index 5a604c9..0be95f9 100644
--- a/m37.py
+++ b/m37.py
@@ -1 +1,2 @@
 v=37
+extra=1
diff --git a/m38.py b/m38.py
index 4fa942b..3005f00 100644
--- a/m38.py
+++ b/m38.py
@@ -1 +1,2 @@
 v=38
+extra=1
diff --git a/m39.py b/m39.py
index a969de4..a39c418 100644
--- a/m39.py
+++ b/m39.py
@@ -1 +1,2 @@
 v=39
+extra=1
diff --git a/m4.py b/m4.py
index 74a3437..d6e0aba 100644
--- a/m4.py
+++ b/m4.py
@@ -1 +1,2 @@
 v=4
+extra=1
diff --git a/m40.py b/m40.py
index 2aa3eb0..917146a 100644
--- a/m40.py
+++ b/m40.py
@@ -1 +1,2 @@
 v=40
+extra=1
diff --git a/m41.py b/m41.py
index 2553c67..928714e 100644
--- a/m41.py
+++ b/m41.py
@@ -1 +1,2 @@
 v=41
+extra=1
diff --git a/m42.py b/m42.py
index 132f944..399ca6e 100644
--- a/m42.py
+++ b/m42.py
@@ -1 +1,2 @@
 v=42
+extra=1
diff --git a/m43.py b/m43.py
index 103fce0..9e4b6ad 100644
--- a/m43.py
+++ b/m43.py
@@ -1 +1,2 @@
 v=43
+extra=1
diff --git a/m44.py b/m44.py
index 06823b9..b2aaf94 100644
--- a/m44.py
+++ b/m44.py
@@ -1 +1,2 @@
 v=44
+extra=1
diff --git a/m45.py b/m45.py
index 0056193..d5d8188 100644
--- a/m45.py
+++ b/m45.py
@@ -1 +1,2 @@
 v=45
+extra=1
diff --git a/m46.py b/m46.py
index 15130ef..0ecff37 100644
--- a/m46.py
+++ b/m46.py
@@ -1 +1,2 @@
 v=46
+extra=1
diff --git a/m47.py b/m47.py
index 5913874..a4b4d71 100644
--- a/m47.py
+++ b/m47.py
@@ -1 +1,2 @@
 v=47
+extra=1
diff --git a/m48.py b/m48.py
index 53cef55..5df1694 100644
--- a/m48.py
+++ b/m48.py
@@ -1 +1,2 @@
 v=48
+extra=1
diff --git a/m49.py b/m49.py
index b3a02bd..c1fe2f9 100644
--- a/m49.py
+++ b/m49.py
@@ -1 +1,2 @@
 v=49
+extra=1
diff --git a/m5.py b/m5.py
index 6dbf679..9bb4027 100644
--- a/m5.py
+++ b/m5.py
@@ -1 +1,2 @@
 v=5
+extra=1
diff --git a/m50.py b/m50.py
index f689fca..afeb9a7 100644
--- a/m50.py
+++ b/m50.py
@@ -1 +1,2 @@
 v=50
+extra=1
diff --git a/m51.py b/m51.py
index bc66eb9..56abc86 100644
--- a/m51.py
+++ b/m51.py
@@ -1 +1,2 @@
 v=51
+extra=1
diff --git a/m52.py b/m52.py
index 98d5da8..4304257 100644
--- a/m52.py
+++ b/m52.py
@@ -1 +1,2 @@
 v=52
+extra=1
diff --git a/m53.py b/m53.py
index d2d3fda..b7d9c6e 100644
--- a/m53.py
+++ b/m53.py
@@ -1 +1,2 @@
 v=53
+extra=1
diff --git a/m54.py b/m54.py
index 268e485..686ef59 100644
--- a/m54.py
+++ b/m54.py
@@ -1 +1,2 @@
 v=54
+extra=1
diff --git a/m55.py b/m55.py
index 8385673..f72f3bc 100644
--- a/m55.py
+++ b/m55.py
@@ -1 +1,2 @@
 v=55
+extra=1
diff --git a/m56.py b/m56.py
index cc8fd8a..aa45f0a 100644
--- a/m56.py
+++ b/m56.py
@@ -1 +1,2 @@
 v=56
+extra=1
diff --git a/m57.py b/m57.py
index e0f00c3..77cd27c 100644
--- a/m57.py
+++ b/m57.py
@@ -1 +1,2 @@
 v=57
+extra=1
diff --git a/m58.py b/m58.py
index 36115f9..41b9f69 100644
--- a/m58.py
+++ b/m58.py
@@ -1 +1,2 @@
 v=58
+extra=1
diff --git a/m59.py b/m59.py
index 31f4971..0734a81 100644
--- a/m59.py
+++ b/m59.py
@@ -1 +1,2 @@
 v=59
+extra=1
diff --git a/m6.py b/m6.py
index 0b9e326..1f778f3 100644
--- a/m6.py
+++ b/m6.py
@@ -1 +1,2 @@
 v=6
+extra=1
diff --git a/m60.py b/m60.py
index faed54d..638018f 100644
--- a/m60.py
+++ b/m60.py
@@ -1 +1,2 @@
 v=60
+extra=1
diff --git a/m7.py b/m7.py
index 0bd4d9b..37189e3 100644
--- a/m7.py
+++ b/m7.py
@@ -1 +1,2 @@
 v=7
+extra=1
diff --git a/m8.py b/m8.py
index f17c82c..3e99f64 100644
--- a/m8.py
+++ b/m8.py
@@ -1 +1,2 @@
 v=8
+extra=1
diff --git a/m9.py b/m9.py
index 243fffa..34ebb5e 100644
--- a/m9.py
+++ b/m9.py
@@ -1 +1,2 @@
 v=9
+extra=1
